src/components/Botton.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] @ambrose-kibet it's not good practice to disable linter checks in your file, kindly fix the linter errors instead of disabling the checks.

Reply:
@danmainah please clarify which boilerplate I should remove . I installed the dependency for the second issue but somehow got the linter error. so if you are aware of any other alternative fixes please share them.